### PR TITLE
Use new product builder for Factory

### PR DESCRIPTION
**Pull request by DimStar77**

### CODE REVIEW

File: commit.sh [hunk 1]
```diff
@@ -6,47 +6,52 @@ test -n "$proj" || proj=Factory
...
+                product=_product
                 ;;
 	Leap:42.*:Ports) arches="ppc64le aarch64"
-            repo="ports"
+                repo="ports"
+                product=_product
                 ;;
 	Factory:PowerPC) arches="ppc64 ppc64le"
-			repo="standard"
+		repo="standard"
```
Comment (jberry-suse) [inline commit.sh]: nit: line uses tabs vs all others are spaces

File: update-tests/testall.sh [hunk 2]
```diff
@@ -2,7 +2,8 @@
 export LC_ALL=C
 
 proj=$1
-(cd ../osc/openSUSE\:$proj/_product/ && osc up )
+product=$2
+(cd ../osc/openSUSE\:$proj/$product/ && osc up )
```
Comment (jberry-suse) [inline update-tests/testall.sh]: nit: might as well fix the unbalanced space as the new one introduced lacks it as well.

**Comment by DimStar77** [comment]
product-builder migration is cancelled for now - a new attempt will be in August - DO NOT MERGE THIS

**Comment by lnussel** [comment]
superseded by #19

**Pull request closed without merging**